packages/manager/modules/hub/src/components/user-panel/user-infos/template.html

```diff
@@ -0,0 +1,74 @@
+<div class="manager-hub-user-infos">
+    <span
+        class="manager-hub-user-infos_initials"
+        data-ng-class="{
+            'manager-hub-user-infos_initials-centered': !$ctrl.isExpanded,
+            'float-left': $ctrl.isExpanded
+          }"
+        data-ng-if="$ctrl.me"
+        data-ng-bind="$ctrl.getNameInitials()"
+    >
+    </span>
+    <p class="oui-chip" data-ng-if="!$ctrl.isExpanded">
+        <oui-skeleton
+            class="mx-auto"
+            data-ng-if="!$ctrl.supportLevel"
+            data-size="xs"
+        ></oui-skeleton>
+        <small
+            data-ng-if="$ctrl.supportLevel"
+            data-ng-bind="('hub_user_support_level_' + $ctrl.supportLevel.level) | translate"
+        ></small>
+    </p>
+    <p data-ng-if="!$ctrl.me">
+        <oui-skeleton class="mx-auto" data-size="s"></oui-skeleton>
+    </p>
+    <div
+        class="manager-hub-user-infos_user-name mb-2"
+        data-ng-class="{
+         'manager-hub-user-infos_user-name__expanded': $ctrl.isExpanded,
+       }"
+        data-ng-if="$ctrl.me"
+    >
+        <button
+            type="button"
+            class="btn btn-link"
+            data-ng-click="$ctrl.isExpanded = !$ctrl.isExpanded"
+        >
+            <span data-ng-bind="$ctrl.getDisplayName()"></span>
+            <span
+                class="oui-icon"
+                data-ng-class="{
+                    'oui-icon-chevron-up': !$ctrl.isExpanded,
+                    'oui-icon-chevron-down': $ctrl.isExpanded,
+                  }"
+            >
+            </span>
+        </button>
+    </div>
+    <div data-ng-if="!$ctrl.isExpanded">
+        <p data-ng-if="!$ctrl.me">
+            <oui-skeleton class="mx-auto" data-size="m"></oui-skeleton>
+            <oui-skeleton class="mx-auto" data-size="s"></oui-skeleton>
+        </p>
+        <p data-ng-if="$ctrl.me">
```

Comment:
`small` has a semantic meaning, it's used for small prints, use CSS instead to have smaller text